src/types/ethr.rs

```diff
@@ -178,12 +178,14 @@ impl EthrBuilder {
     ///  When resolving DIDs with publicKey identifiers, if the controller (owner) address is different from the corresponding address of the publicKey, then the #controllerKey entry in the verificationMethod array MUST be omitted.
     ///
     ///  referecne: [spec](https://github.com/decentralized-identity/ethr-did-resolver/blob/master/doc/did-method-spec.md#controller-changes-didownerchanged)
-    pub fn owner_event(&mut self, event: DidownerChangedFilter) -> Result<()> {
+    pub fn owner_event(&mut self, event: DidownerChangedFilter) -> Result<bool> {
```

Comment:
this should close #10 , but we need a test for owner deactivation, if you want to add that or open a new ticket

Reply:
yes, I'll update #10 ; I wan't sure how to test it, though.